.Zip'ped binary for Win users

JustasB opened the pull request:
So they don't need to install SVN (to get jnml) to run a NeuroML/LEMS model.

pgleeson commented:
It may be better to have regular binary (pre)releases on Github here https://github.com/NeuroML/jNeuroML/releases rather than include a compiled file in the repo. The README can be updated appropriately...

JustasB commented:
Updated the readme, and removed the bins from the bin folder. Created release on my fork, but it won't transfer with the pull request. Cannot create new release on main project as I don't have access. Consider replicating the release on the main branch or giving me access to the repo.

pgleeson commented:
I've some more tests to do locally & I'll generate a release locally

pgleeson commented:
Haven't had a chance to make a release before CNS. This should probably be coordinated with a stable release of PyNeuroML too, using the same binary. All this will need to be tested with all relevant OSB models... After CNS...

pgleeson commented:
See #26. Updated docs to point to binary release.
